### DISCUSSION TITLE
Program broke when creating a mesh using pv.PolyData()

### DISCUSSION THREAD

**Discussion by zhengliuX** [pyvista/pyvista#2607] [Q&A]
Hi, I have the vertex and faces(the connection of the vertex) of a brain, both in np.array. 
Here is my code
```
def load_data():
    '''load the vertex and face'''
    ***
    return all_ver, all_face

import pyvista  as pv
plotter = pv.Plotter(window_size=(500, 300), notebook=False, line_smoothing=True)
mesh = pv.PolyData(all_ver, all_face)
plotter.add_mesh(mesh, cmap='Spectral', style='surface')
```
When running ```mesh = pv.PolyData(all_ver, all_face)```, the program broke, and it shows ```Process finished with exit code -1073741819 (0xC0000005)```.
The OS is Win10 and the shape of all_ver is (318750, 3), and the shape of all_face is (637492, 3), using Python3.7.12

**Comment by zhengliuX** [comment]
Here are the data
https://drive.google.com/file/d/1ojGT_DL_6kpjeeuz-djJiZYcr8crba7d/view?usp=sharing
https://drive.google.com/file/d/1j6CdcHzsj-8CSnF1n_JGOhnPa_ZMpxiD/view?usp=sharing

**Comment by zhengliuX** [comment]
Solved. Forget to add the number of vertex in each connection

**Comment by RichardScottOZ** [comment]
Good! Close this one?

**Comment by zhengliuX** [comment]
> Good! Close this one?

OK!
